server_ui/register_namespace.go

```diff
@@ -58,12 +58,15 @@ func keyIsRegistered(privkey jwk.Key, url string) (keyStatus, error) {
 		return noKeyPresent, err
 	}
 
-	req, err := http.NewRequest("GET", url+"/.well-known/issuer.jwks", nil)
+	req, err := http.NewRequest("GET", url, nil)
+
 	if err != nil {
 		return noKeyPresent, err
 	}
 
 	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("prefix", prefix)
```

Comment:
I couldn't figure if/where we check this header. Why do we set it here? Technically I think the IETF has [deprecated](https://datatracker.ietf.org/doc/html/draft-saintandre-xdash-00) the recommendation of prefixing custom headers with `X-`, but we since every other custom Pelican header we use starts with `X-Pelican-`, we should be consistent.

Reply:
This is checked in `namespace_registry/registry.go` in `dbGetNamespace`